packages/mcp-server/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "batch-upgrade-npm-packages-mcp",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "MCP server for batch upgrading npm packages across repositories",
+  "main": "dist/index.js",
+  "bin": {
+    "batch-upgrade-mcp": "dist/index.js"
```

Comment:
i believe the "right" way to do this is we hide this file in git, but it gets built before we publish so that it's in npm.